source/layouts/_navbar.erb

```diff
@@ -9,6 +9,9 @@
           <li>
             <a href="/index.html#interests">Interests</a>
           </li>
+          <li>
+            <a href="/index.html#blog">Blog</a>
```

Comment:
I can't believe I've never noticed this before, but can we drop the `/index.html` from all the nav elements? It makes for a very jarring effect when clicking on the items. The anchor tag by itself provides a much smoother transition.

Reply:
The navbar partial is used on other pages than index as well, so it needs some more work than just dropping `index.html`. I didn't notice jarring when using the links, but there's apparent reflow effect caused by the calendar widget, when scrolling or navigating using links before it's loaded. I added `min-height` to that row to reduce it.